usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -125,6 +136,14 @@ def signal_handler(signal, frame):
         load_file_c(submission_attributes, self.db_cursor, certified_award_financial)
         logger.info(f"Finished loading File C data, took {datetime.now() - start_time}")
 
+        if self.skip_final_of_fy_calculation:
+            logger.info("Skipping final_of_fy calculation as requested.")
+        else:
+            logger.info("Updating final_of_fy")
+            start_time = datetime.now()
```

Comment:
What about that fancy new `now()` function?

Reply:
That would be awesome, but this largely copy paste code.  I didn't want to fix EVERYthing.  I'd never get the PR done... which is already a risk.  😛